### ISSUE TITLE
Database error logs when I enter Explore view

### ISSUE THREAD

**Issue by ericpgreen2** [rilldata/rill-developer#574]
Two of these "Parser Error" logs show up in my console every time I enter the Explore view.
![image](https://user-images.githubusercontent.com/14206386/179326635-9200bf87-6b56-46c7-98bf-ff761c7c631b.png)

I'm using the AdBids dataset.

This is my model:
```
select epoch_ms(timestamp) as timestamp, bid_price, domain, publisher
from AdBids
```

I use the "quick metrics" button to create a sole `count(*)` measure.

**Comment by AdityaHegde** [comment]
These should be fixed in #532 . I am not getting this error anymore. Please confirm if you still get it @ericpgreen2

**Comment by ericpgreen2** [comment]
Great, I'm not seeing these anymore. I'll close 👍
